dss_datamover/logger.py

```diff
@@ -163,10 +162,15 @@ def logging(self, queue, lock, stop_logging):
                         (message_level, message_value) = message
                         print("{}: {}".format(LOGGING_LEVEL[message_level], message_value))
                         fh.write(str(time.ctime()) + ": " + LOGGING_LEVEL[message_level] + ": " + message_value + "\n")
+                    elif message is None:
+                        stop_flag = True
```

Comment:
question here. If the logging queue is ever empty, does that mean we want to stop the logger? Could it be possible for the queue to be empty at one point in time and then become populated at another time with more messages? Just wanted to verify this is the behavior we want in this case

Reply:
None message is the last one generated by the master_application/client_application after stopping all the processes (aka workers/tasks).